app/src/main/java/com/kickstarter/ApplicationModule.java

```diff
@@ -673,7 +694,7 @@ ExperimentsClientType provideOptimizely(final @ApplicationContext @NonNull Conte
     }
 
     final OptimizelyManager optimizelyManager = OptimizelyManager.builder()
-      .withSDKKey(optimizelyEnvironment.getSdkKey())
+      .withSDKKey("777777")
```

Comment:
Just curious, what is this change doing?

Reply:
My bad! I was testing how optimizely behaves with incorrect/onactive credentials, I'll revert that back